frontend/src/pages/acceleratorProfiles/screens/list/AcceleratorProfilesTable.tsx

```diff
@@ -12,26 +14,63 @@ type AcceleratorProfilesTableProps = {
 
 const AcceleratorProfilesTable: React.FC<AcceleratorProfilesTableProps> = ({ accelerators }) => {
   const navigate = useNavigate();
+  const [searchType, setSearchType] = React.useState<SearchType>(SearchType.NAME);
+  const [search, setSearch] = React.useState('');
+  const filteredAcceleratorProfiles = accelerators.filter((accelerator) => {
+    if (!search) {
+      return true;
+    }
+
+    switch (searchType) {
+      case SearchType.NAME:
+        return accelerator.spec.displayName.toLowerCase().includes(search.toLowerCase());
+      case SearchType.IDENTIFIER:
+        return accelerator.spec.identifier.toLowerCase().includes(search.toLowerCase());
+      default:
+        return true;
+    }
+  });
+
+  const resetFilters = () => {
+    setSearch('');
+  };
+
+  const searchTypes = React.useMemo(() => [SearchType.NAME, SearchType.IDENTIFIER], []);
 
   return (
     <Table
       id="accelerator-profile-table"
       enablePagination
-      data={accelerators}
+      data={filteredAcceleratorProfiles}
       columns={columns}
-      emptyTableView={'No projects match your filters.'}
+      emptyTableView={<DashboardEmptyTableView onClearFilters={resetFilters} />}
```

Comment:
Can you inline this? Just like what you've done for `onSearchTypeChange` and `onSearchValueChange`.